pkg/policy/repository.go

```diff
@@ -428,6 +428,37 @@ func (p *Repository) DeleteByLabelsLocked(labels labels.LabelArray) (ruleSlice,
 	return deletedRules, p.GetRevision(), deleted
 }
 
+// GetAllSelectors returns all of the EndpointSelectors and FQDNSelectors from
+// the given slice of rules.
+func (r ruleSlice) GetAllSelectors() ([]api.EndpointSelector, []api.FQDNSelector) {
+	epSels := make([]api.EndpointSelector, 0)
+	fqdnSels := make([]api.FQDNSelector, 0)
+	for _, rr := range r {
+		rEpSels, rFqdnSels := rr.GetAllSelectors()
+		epSels = append(epSels, rEpSels...)
+		fqdnSels = append(fqdnSels, rFqdnSels...)
+	}
+	return epSels, fqdnSels
+}
+
+// GetAllSelectors returns all of the EndpointSelectors and FQDNSelectors from
+// the given rule.
+func (r *rule) GetAllSelectors() ([]api.EndpointSelector, []api.FQDNSelector) {
```

Comment:
This is only used from the un-used function above, consider removing the dead code.